pkg/bmc/ipmi.go

```diff
@@ -73,6 +75,16 @@ func (a *ipmiAccessDetails) DriverInfo(bmcCreds Credentials) map[string]interfac
 	return result
 }
 
+// NodeProperties returns a data structure to return details of
+// the host, including the boot mode. This will be used later to
+// instruct ironic to use specific boot mode
+func (a *ipmiAccessDetails) NodeProperties() map[string]interface{} {
```

Comment:
I think we should have an option here, to discriminate between ipv4/ipv6. And ipv6 should be uefi. Please send feedback on this question.

Reply:
OH! Just saw this comment. I think in practicality terms, yes IPv6 seems to always imply UEFI. I _suspect_ in reality, some hardware supports bios mode with IPv6 and I kind of worry about abstracting additional logic to do things automatically into particular configurations. I guess I'm unsure. @dtantsur @derekhiggins Do you guys have any thoughts on this?

Reply:
Which address matters, the one the BMC uses or the one the host is going to receive? How do we know the latter when we're going to call this method?